{{cookiecutter.project_slug}}/local.yml

```diff
@@ -29,6 +30,7 @@ services:
       context: .
       dockerfile: ./compose/production/postgres/Dockerfile
     image: {{ cookiecutter.project_slug }}_production_postgres
+    container_name: database
```

Comment:
Hi,  I just changed the container names to match the service names.  

I call docker-compose indirectly through a makefile. Using container_name has been nice for me, so I thought it might be helpful for others. 

But I also didn't know about using docker-compose that way!

> So it looks like we wouldn't be able to run 2 (or more) Django containers? I assume it might be a common use case for folks on production...

This is why I only put the change in local.yml (but since I don't use multi-container services, I am not sure this is actually a solution).

It is a trade off! I am not discouraged if you decide it's too much of a trade off.